### PR TITLE
WRP-4092: IncrementSlider - refactor utils.js

### PR DESCRIPTION
**Pull request by adrian-cocoara-lgp**
Enact-DCO-1.0-Signed-off-by: Adrian Cocoara adrian.cocoara@lgepartner.com

### Checklist

* [x] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [ ] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [ ] At least one test case is included for this feature or bug fix
* [x] Documentation was added or is not needed
* [x] Documentation was verified or is not changed
* [x] UI test was passed or is not needed
* [x] Screenshot test was verified or is not needed
* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)
`IncrementSlider/utils.js` contains some copied functions form `Slider/utis.js` which are not used in the implementation of component, so they were removed.

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)

### Additional Considerations
[//]: # (How should the change be tested?)
[//]: # (Are there any outstanding questions?)
[//]: # (Were any side-effects caused by the change?)

### Links
[//]: # (Related issues, references)
WRP-4092

### CODE REVIEW

**Review by daniel-stoian-lgp — APPROVED** [review]

**Comment by daniel-stoian-lgp** [comment]
note for the next reviewer: since the deleted functions are private, we think a changelog is not necesary. Please correct us if we are wrong

File: IncrementSlider/utils.js [hunk 1]
```diff
@@ -16,24 +15,7 @@ const calcStep = (knobStep, step) => {
...
-const isNotMax = (ev, {value, max}) => {
-	return value !== max;
-};
-
-const isNotMin = (ev, {min, value = min}) => {
-	return value !== min;
-};
-
-const emitChange = (direction) =>  adaptEvent(
+export const emitChange = (direction) =>  adaptEvent(
```
Comment (juwonjeong) [inline IncrementSlider/utils.js:18]:
```suggestion

export const emitChange = (direction) =>  adaptEvent(
```
Please add a new line. Other changes look good to me.

**Comment by juwonjeong** [comment]
I agree with @daniel-stoian-lgp that this fix does not need to add CHANGELOG.

**Review by juwonjeong — APPROVED** [review]
LGTM

**Pull request merged**